raw_inference.py

```diff
@@ -1,111 +1,249 @@
+# import os
```

Comment:
@nguyennm1024 : If you don't use this code anymore, please remove it instead of just commenting out.